src/views/includes/display-address.njk

```diff
@@ -0,0 +1,25 @@
+{% import 'includes/macros.njk' as macros %}
```

Comment:
we already have a formatAddress macro in macros.njk, could that be used instead?

Reply:
Not sure what you mean?

Reply:
we already have a function to format an address for display, it is used on the 'choose address' page where we list addresses with a radio button after entering the postcode:
https://github.com/companieshouse/limited-partnerships-web/blob/main/src/views/pages/address/choose-address-form.njk#L7
Just wondering if we can call that function in the macro instead of having this display-address.njk file

Reply:
Works with the marco.